Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -725,16 +725,83 @@ int _CGPathPointCountForElementType(CGPathElementType type) {
     return pointCount;
 }
 
+class _CGPathApplySink
+    : public Microsoft::WRL::RuntimeClass<Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::RuntimeClassType::ClassicCom>,
+                                          IAgileObject,
+                                          Microsoft::WRL::FtmBase,
+                                          ID2D1SimplifiedGeometrySink> {
+public:
+    _CGPathApplySink(_In_ void* info, _In_ CGPathApplierFunction function) : m_info(info), m_pathApplierFunction(function) {
+    }
+
+    STDMETHOD_(void, AddBeziers)(const D2D1_BEZIER_SEGMENT* beziers, UINT bezierCount) {
+        for (UINT i = 0; i < bezierCount; ++i) {
```

Comment:
It's the recommended typing from D2D in their instructions on how to implement the custom geometry sink. I would prefer to match their documentation in this case.

---
In reply to: [91124025](https://github.com/Microsoft/WinObjC/pull/1507#discussion_r91124025) [](ancestors = 91124025)